src/main/webapp/js/email-browser.js

```diff
@@ -357,9 +528,22 @@ var jasig = jasig || {};
             dataFunction: getEmailFunction(that, that.options.folderName),
             dataLengthFunction: function() { return account.accountSummary ? account.accountSummary.totalMessageCount : 0; }
         };
-
         that.pager = unicon.batchedpager(that.locate("emailList"), batchOptions);
 
+        // Notify the server of changes to pageSize so they can be remembered and update the container min-height class
+        that.pager.pager.events.initiatePageSizeChange.addListener(function(newPageSize) {
+            // console.log("newPageSize: "+ newPageSize);
+            /* if #right-content-email has .emailpreview-container-min-height-XX class remove it */
```

Comment:
@ChristianMurphy 
it will be #right-content-email.min-height-XX
according to my comment in [theme.less#L343](https://github.com/cousquer/email-preview/blob/3a68f3413c79609733d7c2ab7d9c2977980302f3/src/main/webapp/less/includes/theme.less#L343)